examples/helia-101/102-cid-profiles.js

```diff
@@ -0,0 +1,52 @@
+import { createReadStream } from 'node:fs'
+import { unixfs } from '@helia/unixfs'
+import { createHelia } from 'helia'
+import { fixedSize } from 'ipfs-unixfs-importer/chunker'
+import { balanced } from 'ipfs-unixfs-importer/layout'
+
+
+// set up
+
+const helia = await createHelia()
+const fs = unixfs(helia)
+
+// using a local copy of https://www.gutenberg.org/files/2600/2600-h/2600-h.htm , curled to this directory but not checked in
```

Comment:
There's an irony here. If this file changes for whatever reason this will end up having a different CID.

Reply:
that's why i used a project gutenberg URL, the contents are pretty stable and they never update formatting!